src/ast.rs

```diff
@@ -7450,6 +7458,26 @@ impl Parse for SpecialName {
 
                 Ok((SpecialName::JavaResource(resource_names), tail))
             }
+            b"GT" => {
+                match tail.peek() {
+                    None => return Err(error::Error::UnexpectedEnd),
+                    Some(b'n') => {
+                        let tail = consume(b"n", tail)?;
+                        let (base, tail) =  Encoding::parse(ctx, subs, tail)?;
+                        Ok((SpecialName::NonTransactionClone(Box::new(base)),
+                            tail,
+                        ))
+                    }
+                    Some(b't') | _ => {
+                        let (_, tail) = tail.split_at(1);
```

Comment:
And (b't', tail) | (_, tail) => { TransactionClone branch}

Also a comment about how libiberty accepts all GT? where ? != n as equivalent to GTt would be good.